### DISCUSSION TITLE
Keyboard Navigation using the Tab Key

### DISCUSSION THREAD

**Discussion by StefanKoell** [AvaloniaUI/Avalonia#13687] [Q&A]
Hi,

I asked this on the telegram channel already but it seemed to got missed.

I'm trying to implement proper keyboard navigation to conform to some accessibility standards. The topic is not really well documented and from what I read, it's kind of cumbersome to setup and then I failed to implement it properly.

On the Menu and MenuItems at least I was able to make tab navigation possible by setting TabNavigation to Continue and each MenuItem's IsTabStop to true.

Then I have two ItemsRepeater which generate ToggleButtons on one level and a set of ToggleButtons a level below (nested repeaters). I'm not able to get tab navigation to work properly and I'm not sure if this is just a bug or if I'm doing something wrong.

Is there any guidance/documentation on how to properly set up this kind of thing. I was actually surprised that it's not enabled by default and that it's so much work to get it to work on a basic level.

Any help would be greatly appreciated. Thanks!

**Comment by timunie** [comment]
I believe this is related to https://github.com/AvaloniaUI/Avalonia/issues/7607. Some was working on it, but the PR got closed.

**Reply by StefanKoell** [reply]
I've seen this as well but I'm not sure it's related. I think the main issue is that there's no documentation and I'm not sure if I set up everything correctly. Maybe I missed something in the docs. Not sure.

**Reply by StefanKoell** [reply]
@timunie since I haven't got any further response, should I file an issue with a repro?

**Reply by timunie** [reply]
Yeah please do so for now. Can't promise anything but having a sample often helps a lot. 

💡: You could also add the sample to src/Sandbox and see if you find the root cause.

**Reply by StefanKoell** [reply]
Opened an issue #14095
